lib/pay/stripe/webhooks/customer_updated.rb

```diff
@@ -9,14 +9,21 @@ def call(event)
           # Couldn't find user, we can skip
           return unless pay_customer.present?
 
+          stripe_customer = pay_customer.customer
+
           # Sync default card
-          if (payment_method_id = pay_customer.customer.invoice_settings.default_payment_method)
+          if (payment_method_id = stripe_customer.invoice_settings.default_payment_method)
             Pay::Stripe::PaymentMethod.sync(payment_method_id, stripe_account: event.try(:account))
 
           else
             # No default payment method set
             pay_customer.payment_methods.update_all(default: false)
           end
+
+          # Sync invoice credit balance and currency
+          if stripe_customer.invoice_credit_balance&.keys.present?
```

Comment:
@nachiket87 - I don't think we need to do this call to `.keys` here. It feels like we just want to check that the resulting object isn't empty. So I think we could just do `stripe_customer.invoice_credit_balance.present?` instead. I think it reads better additionally i.e. "Does the customer have an invoice credit balance?" versus "Does the customer have invoice credit balance keys?"

As an example:
```ruby
stripe_customer = OpenStruct.new(invoice_credit_balance: {"usd" => 12345})
=> #<OpenStruct invoice_credit_balance={"usd"=>12345}>
stripe_customer.invoice_credit_balance.present?
=> true

stripe_customer = OpenStruct.new(invoice_credit_balance: {})
=> #<OpenStruct invoice_credit_balance={}>
stripe_customer.invoice_credit_balance.present?
=> false
```

Reply:
Thanks @cjilbert504 I've refactored it in 968f6767142c56558d0161ea0825886b6e1b95ea

Reply:
I ended up saving the whole `invoice_credit_balance` just in case there's a need for someone to access the other currencies. This will be the safest / most future proof.